compiler/src/iree/compiler/Dialect/HAL/IR/HALOps.td

```diff
@@ -811,12 +812,6 @@ def HAL_ElementTypeOp : HAL_PureOp<"element_type", [
     `:` type($result)
   }];
 
-  let builders = [
```

Comment:
I had to remove this here. I dont know why. It was giving me a duplicate definition error.

Reply:
weird